Fix images in evaluation notebook

Hakimovich99 opened the pull request:
a few changes brought by Jan to the notebooks.

RobbeSneyders commented:
Hi Hakim, these are the old images being added again, which don't reflect the true pipeline. Let's stick to the ones I added. Also really don't like the gif at the end.

Hakimovich99 commented:
> Hi Hakim, these are the old images being added again, which don't reflect the true pipeline. Let's stick to the ones I added. Also really don't like the gif at the end.

Oh alright alright 

> Hi Hakim, these are the old images being added again, which don't reflect the true pipeline. Let's stick to the ones I added. Also really don't like the gif at the end.

Ah okok, sorry I took them from Jan but didn't know there were other new images. But I actually don't see any image in the notebook from the main branch. Is it only me?

RobbeSneyders commented:
They seem to be broken indeed, my bad. I posted the excalidraw with the latest images in chat. Can you update them?

Hakimovich99 commented:
> They seem to be broken indeed, my bad. I posted the excalidraw with the latest images in chat. Can you update them?

Sure! Can you pin me pls? Can't find it haha

RobbeSneyders commented on src/evaluation.ipynb:
```diff
@@ -119,14 +111,14 @@
    "outputs": [],
    "source": [
     "!docker compose version\n",
-    "!docker info >> /dev/null && echo \"Docker running\""
```
Don't know why this was removed, but I don't think we need this output.

RobbeSneyders commented on src/evaluation.ipynb:
```diff
@@ -498,16 +469,26 @@
+    "And of course feel free to give us a [**star** ⭐](https://github.com/ml6team/fondant) if you like what we are doing!\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"../art/feedback_is_important.webp\"/>\n",
```
Would still remove this :)
